src/Interfaces/LazySet.jl

```diff
@@ -682,3 +683,44 @@ end
 
 isoperationtype(::LazySet) = error("`isoperationtype` cannot be applied to " *
                                    "a set instance; use `isoperation` instead")
+
+"""
+    isequivalent(X::LazySet, Y::LazySet)
+
+Return whether two LazySets are equal in the mathematical sense, i.e. equivalent.
+
+### Input
+
+- `X` -- any `LazySet`
+- `Y` -- another `LazySet`
+
+### Output
+
+`true` iff `X` is equivalent to `Y`.
+
+## Algorithm
+
+First, the check `X == Y` is performed which returns `true` if and only if the given sets are of the same type,
+and have the same values (modulo floating-point tolerance). Otherwise, the double inclusion check `X ⊆ Y && Y ⊆ X` is
+computed.
```

Comment:
```suggestion
used.
```
I would say that a test is not computed.